deps/rabbit/src/rabbit_stream_queue.erl

```diff
@@ -229,7 +230,7 @@ begin_stream(#stream_client{readers = Readers0} = State0,
             {error, no_local_stream_replica_available};
         _ ->
 
-            {ok, Seg0} = osiris:init_reader(LocalPid, Offset),
+            {ok, Seg0} = osiris:init_reader(LocalPid, Offset, 'channel'),
```

Comment:
I think we should use the ?MODULE as the tag here as we may use the queue type inside processes that are not channels.